tests/normal.t.pl

```diff
@@ -28,10 +28,10 @@ sub runTests
     my $hardlinks = 1;
     if($mode eq 'standard')
     {
-        &mount("--standard");
+        &mount("--standard -o direct_io");
     } elsif($mode eq 'paranoia')
     {
-        &mount("--paranoia");
+        &mount("--paranoia -o direct_io");
```

Comment:
Well, when you look at the test which fails with FreeBSD (58/104, see #315), the fact that it fails is quite expected :
- the _fuse_ file is created, FreeBSD puts it into its cache, then it is closed ;
- the underlying file is corrupted putting some garbage into it ;
- the _fuse_ file is reopened, and fully read from the cache.
Of course the cache has not been updated when underlying file has been modified, as I'm not sure it can work this way.

I think this behaviour is quite "expected". Don't you think so ?
Should we update the test so that it will fail without `direct_io` ?